### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Perform local run_Keyboard: Disconnect, Restore Default Layout, Pin Tab, Soft wrap, Scroll to End controls, Print, Settings are not accessible using keyboard

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/azure-tools-for-java#3589]
GitHub Tags: #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #Keyboard; #MAS2.1.1; 

[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.1
Build #IU-192.6262.58

**OS Details:**
OS: Windows (Version 1903, Build: 18362.113)

**MAS Violated:** 2.1.1

**Note:
Similar user is unable to access filter, mute break points, settings, get thread dump, pin tab controls in result window of the debugging section(Perform Local Debugging)**

**Repro Steps:**
1)Launch IntelliJ IDEA in Light Theme
2)Tab till create new project-->Azure Spark/HDInsight-->Spark project with Samples(Scala) 
3)Tab till next button and press enter
4)Fill all the details correctly and Tab till finish button and press enter
5)Tab till <Project Name>-->src-->main-->scala-->sample-->SparkCore_wasbloTest and open it
6)Open the context menu using keyboard and navigate to '[Spark Job]XXX' to perform local run
7)Verify user is able to access  Disconnect, Restore Default Layout, Pin Tab, Soft wrap, Scroll to End, Print, Settings controls using keyboard or not

**Actual Result:**
 Disconnect, Restore Default Layout, Pin Tab, Soft wrap, Scroll to End, Print, Settings controls are not accessible using keyboard

**Expected Result:**
 Disconnect, Restore Default Layout, Pin Tab, Soft wrap, Scroll to End, Print, Settings controls should be accessible using keyboard

**User Impact:**
Keyboard users will be unable to access the controls if keyboard focus is not going there

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachments:**
![Bug_3589_local debug](https://user-images.githubusercontent.com/42638450/65412171-47f93900-de0c-11e9-8899-1fac4668bc64.png)
![Bug_3589_local run](https://user-images.githubusercontent.com/42638450/65412202-5e06f980-de0c-11e9-893c-f879c3c65ade.png)

**Comment by t-rufang** [comment]
1. As for the first issue, focus is not going to "filter, mute break point, get thread dump, pin tab, settings, more options", this scenario should be supported by IntelliJ.
![Bug_3589_local debug](https://user-images.githubusercontent.com/42638450/65412171-47f93900-de0c-11e9-8899-1fac4668bc64.png)

2. As for the second bug, "Disconnect, Restore Default layout, Pin Tab, Soft wrap, Scroll to End controls, Print, Settings" are not accessible using keyboard, we need to figure out if this should be handled by tool. 
![Bug_3589_local run](https://user-images.githubusercontent.com/42638450/65412202-5e06f980-de0c-11e9-893c-f879c3c65ade.png)

**Comment by v-jiche** [comment]
Still repro.

build:
azure-toolkit-for-intellij-2019.2.develop.1276.10-25-2019

**Comment by t-rufang** [comment]
For disconnect button issue, it's fixed in #3723

**Comment by InduPriya1805** [comment]
Regressed the bug in the below environment
IntelliJ IDEA 2019.2.2
Build #IU-192.6603.28

Now user is able to access "Disconnect" button using shortcut key "Cntrl+Alt+Shift+F2". Please find the video attached for reference.
**Attachment:**
[Bug_3589_disconnect button_fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3857851/Bug_3589_disconnect.button_fixed.pptx)

And the remaining all the controls "Restore Default Layout, Pin Tab, Soft wrap, Scroll to End controls, Print, Settings, filter, mute break point, get thread dump, pin tab, settings, more options" are external to IntelliJ IDEA

**Comment by t-rufang** [comment]
- Importance of experience: Medium, keyboard only users can't navigate to the following buttons in debug tool window: Restore Default Layout, Pin Tab, Soft wrap, Scroll to End, Print, Settings controls.
- Quality of experience: Annoying, for debug feature these actions not the mostly used actions. Without these features user can still debug their application but it's not very convenient. 
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by InduPriya1805** [comment]
Closing this bug as per the discussion with SME

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749692
